ansible/roles/software/datagov-deploy-fgdc2iso/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']
+).get_hosts('all')
+
+
+def test_tomcat(host):
+    tomcat = host.service('tomcat')
+
+    assert tomcat.is_running
+
+
+def test_port(host):
+    assert host.socket("tcp://0.0.0.0:8080").is_listening
+
+
+def test_version(host):
+    cmd = host.run("sh /opt/tomcat/bin/version.sh")
+
+    assert cmd.succeeded
+    assert "Apache Tomcat/9.0.36" in cmd.stdout
+
+
+def test_cron(host):
+    cron = host.file('/etc/cron.d/clean-tmp-files')
+
+    assert cron.exists
+    assert cron.user == 'root'
+    assert cron.group == 'root'
+    assert cron.mode == 0o644
+    assert cron.contains('tmp*-*-*-*-*')
```

Comment:
I think this should be a regular expression.
```suggestion
    assert cron.contains(r'"tmp\*-\*-\*-\*-\*"')
```

Reply:
Not intended to be regex.  Should be exact string matching `tmp*-*-*-*-*`, I am changing it to the following to avoid confusion.
`assert 'tmp*-*-*-*-*' in cron.content_string`